test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1

```diff
@@ -86,4 +86,33 @@ Describe 'Null Representatives' -Tags 'CI' {
             @{ Value = [NullString]::Value; ExpectedCount = 5 }
         )
     }
+
+    Context 'Numeric Comparisons' {
+        BeforeAll {
+            $TestValues = @(
+                @{ Value = { $null } }
+                @{ Value = { [DBNull]::Value } }
+                @{ Value = { [NullString]::Value } }
+                @{ Value = { [AutomationNull]::Value } }
+            )
+        }
+
+        It '<Value> should be considered greater than or equal to 0' -TestCases $TestValues {
+            param($Value)
+
+            0 -ge $Value.InvokeReturnAsIs() | Should -BeTrue
```

Comment:
Any particular reason `0` is left hand side unlike the tests below?  For completeness, it seems `-le` should be covered.

Reply:
Not that I can recall. Will add the `-le` test as well. 🙂

Reply:
Addressed and pushed new commits. Also rebased since this branch is pretty old. 🙂